runtime/oti/jitserver_api.h

```diff
@@ -32,8 +32,32 @@ extern "C" {
 struct JITServer; /* Forward declaration */
 
 typedef struct JITServer {
+	/**
+	 * Starts an instance of JITServer.
+	 *
+	 * @param jitServer pointer to the JITServer interface
+	 *
+	 * @returns JITSERVER_OK on success, else negative error code
+	 */
 	int32_t (* startJITServer)(struct JITServer *);
+	/**
+	 * Wait for JITServer to terminate.
+	 *
+	 * @param jitServer pointer to the JITServer interface
+	 *
+	 * @returns JITSERVER_OK on success, else negative error code
+	 */
 	int32_t (* waitForJITServerTermination)(struct JITServer *);
+	/**
+	 * Stops the JITServer.
+	 *
+	 * @param jitServer double pointer to the JITServer interface
+	 *
+	 * @returns JITSERVER_OK on success, else negative error code
+	 *
+	 * @note on return *jitServer is set to NULL
+	 */
+	int32_t (* destroyJITServer)(struct JITServer **);
```

Comment:
Now that it's clear this frees the resources, I'd like to see it called by the jitserver launcher.  It can be called after 1 waitForJITServerTermination` returns

Reply:
Added the call to `destroyJITServer` in jitserver launcher.